.build/DSC/ConfigData.build.ps1

```diff
@@ -53,7 +56,7 @@ Param (
         $Global:Datum = New-DatumStructure -DefinitionFile $DatumDefinitionFile
         
 
-        $Global:ConfigurationData = Get-FilteredConfigurationData -Environment $Environment -FilteredNode $FilteredNode -Datum $Datum
+        $Global:ConfigurationData = Get-FilteredConfigurationData -Environment $Environment -Filter $Filter -FilteredNode $FilteredNode -Datum $Datum
 }
```

Comment:
You could have replaced the FilterNode, to avoid feature overlap. Never mind, will do another time.

Reply:
Oh, you're right. Should have done that^^. Glad you like it!